src/components/documentManager.ts

```diff
@@ -0,0 +1,611 @@
+            );
+        else return null;
+    }
+
+    /**
+     * Get url for logo image for a given item
+     *
+     * @param {BaseItemDto} item to look up
+     * @returns {string | null} url to logo image
+     */
+    private static getLogoUrl(item: BaseItemDto): string | null {
+        if (item.ImageTags?.Logo)
+            return JellyfinApi.createUrl(
+                `Items/${item.Id}/Images/Logo/0?tag=${item.ImageTags.Logo}`
+            );
+        else if (item.ParentLogoItemId && item.ParentLogoImageTag)
+            return JellyfinApi.createUrl(
+                `Items/${item.ParentLogoItemId}/Images/Logo/0?tag=${item.ParentLogoImageTag}`
+            );
+        else return null;
+    }
+
+    /**
+     * This fucntion takes an item and shows details about it
+     * on the details page. This happens when no media is playing,
+     * and the connected client is browsing the library.
+     *
+     * @param {BaseItemDto} item to show information about
+     */
+    public static showItem(item: BaseItemDto): void {
+        // stop cycling backdrops
+        this.clearBackdropInterval();
+
+        this.setAppStatus('details');
+        this.setWaitingBackdrop(item);
+
+        this.setLogo(this.getLogoUrl(item));
+        this.setOverview(item.Overview ?? null);
+        this.setGenres(item?.Genres?.join(' / ') ?? null);
+        this.setDisplayName(item);
+
+        this.setMiscInfo(item);
+
+        const detailRating = document.getElementById('detailRating');
+        if (detailRating) {
+            detailRating.innerHTML = this.getRatingHtml(item);
+        }
+
+        const playedIndicator = document.getElementById('playedIndicator');
+
+        if (playedIndicator) {
+            if (item?.UserData?.Played) {
+                playedIndicator.style.display = 'block';
+                playedIndicator.innerHTML =
+                    '<span class="glyphicon glyphicon-ok"></span>';
+            } else if (item?.UserData?.UnplayedItemCount) {
+                playedIndicator.style.display = 'block';
+                playedIndicator.innerHTML = item.UserData.UnplayedItemCount.toString();
+            } else {
+                playedIndicator.style.display = 'none';
+            }
+        }
+
+        let detailImageUrl = this.getPrimaryImageUrl(item);
+
+        if (
+            item?.UserData?.PlayedPercentage &&
+            item?.UserData?.PlayedPercentage < 100 &&
+            !item.IsFolder
+        ) {
+            this.setHasPlayedPercentage(false);
+            this.setPlayedPercentage(item.UserData.PlayedPercentage);
+
+            if (detailImageUrl != null)
+                detailImageUrl +=
+                    '&PercentPlayed=' +
+                    item.UserData.PlayedPercentage.toString();
+        } else {
+            this.setHasPlayedPercentage(false);
+            this.setPlayedPercentage(0);
+        }
+
+        this.setDetailImage(detailImageUrl);
```

Comment:
This is the most central code that is responsible for the image on the left hand in the details page. Apparently this image is about half size of what it should be in some cases.

Reply:
Okay I found a css difference between stable and this, turns out there was a css messup.